src/main/java/io/airlift/jodabridge/JdkBasedDateTimeZone.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.jodabridge;
+
+import org.joda.time.DateTimeZone;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.zone.ZoneOffsetTransition;
+import java.time.zone.ZoneRules;
+import java.util.Objects;
+
+public class JdkBasedDateTimeZone
+        extends DateTimeZone
+{
+    private final ZoneRules zoneRules;
+
+    JdkBasedDateTimeZone(String zoneId)
```

Comment:
`private`

Reply:
The visibility needs to be at least package-private.

Reply:
Ah, I misread the code and thought it was constructed here.